### DISCUSSION TITLE
The OrderedVocab you are attempting to save contains a hole for index 5024, your vocabulary could be corrupted !

### DISCUSSION THREAD

**Discussion by jackyin68** [coqui-ai/TTS#3244] [General]
While fine tune xttsv2, using zh-cn language, some information below appear.
This is normal or how to fix it.
==========================================================================>
> Sampling by language: dict_keys(['zh-cn'])
The OrderedVocab you are attempting to save contains a hole for index 5024, your vocabulary could be corrupted !

**Comment by Cyl0nius** [comment]
Same here with German (de) language.
How to fix this?
